airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/utils/connector_runner.py

```diff
@@ -208,6 +210,18 @@ async def call_read_with_state(
             enable_caching=enable_caching,
         )
 
+    async def do_setup(self, dagger_client: dagger.Client, container: dagger.Container, config: SecretDict, setup_script_path: Path):
```

Comment:
I find it confusing to add this logic to the ConnectorRunner class. We're not running command against the Connector, we're running an arbitrary  script which has not direct relationship with the connector execution right?